x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/build_bulk_body.ts

```diff
@@ -121,7 +122,11 @@ export const buildBulkBody = (
       ),
       ...additionalAlertFields({
         ...mergedDoc,
-        _source: { ...mergedDoc._source, ...validatedEventFields },
+        _source: {
+          ...validatedSource,
+          ...validatedEventFields,
+          threshold_result: thresholdResult,
```

Comment:
@vitaliidm I had to add threshold results here, because that object is being stripped and is not part of `validatedSource`. We expect `threshold_result` inside `additionalAlertFields` though. Let me know what you think about this addition.

Discovered this due to these build failures https://buildkite.com/elastic/kibana-pull-request/builds/219594